engine/src/main/java/org/terasology/logic/ai/HierarchicalAISystem.java

```diff
@@ -125,17 +125,17 @@ private void loop(EntityRef entity, LocationComponent location,
 
             // if our AI is aggressive or hunter go and hunt player else run away
             // if wild
-            if (ai.aggressive) {
+            /*if (ai.aggressive) {
```

Comment:
I'd say use the blame command.  I wonder if you can do version-by-version blames on GitHub.